### DISCUSSION TITLE
Meshcentral messaging doesn't work

### DISCUSSION THREAD

**Discussion by Avessalom608** [amidaware/tacticalrmm#1856] [Q&A] [answered]
After updating from version 0.18.0 to 0.18.2, internal chat via MeshCentral stopped working, while the RDP connection works without any problems. I didn't find anything in the logs.

Waiting for the other side...
The connection has been interrupted.

I looked at the logs at the address /rmm/api/tacticalrmm/tacticalrmm/private/log/error.log

**Comment by silversword411** [comment, marked as answer]
I just tested with mesh sync active. Just works.

Might need to turn off sync, then turn it back on again after waiting a couple mins. See https://docs.tacticalrmm.com/mesh_integration/#turn-off-the-sync-feature
